packages/light-client-experimental/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@substrate/light-client-experimental",
+  "version": "0.0.1",
+  "author": "Ryan Lee (https://github.com/ryanleecode)",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/paritytech/substrate-connect.git"
+  },
+  "license": "MIT",
+  "exports": {
+    ".": "./dist/index.mjs",
+    "./smoldot": "./dist/smoldot/index.mjs",
+    "./json-rpc-provider": "./dist/json-rpc-provider/index.mjs"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc --noEmit && tsup-node src/index.ts src/smoldot/index.ts src/json-rpc-provider/index.ts --clean --sourcemap --platform neutral --target=es2020 --format esm --dts",
+    "dev": "pnpm build --watch",
+    "lint": "prettier --check README.md \"src/**/*.{js,jsx,ts,tsx,json,md}\""
+  },
+  "prettier": {
+    "printWidth": 80,
+    "semi": false,
+    "trailingComma": "all"
+  },
+  "devDependencies": {
+    "@effect/experimental": "^0.15.18",
```

Comment:
DQ (I'll have lots of these): Do we need this "experimental" package (always a word I'd rather avoid seeing :D), and what does it give over "effect"?